### DISCUSSION TITLE
Draging pinned image

### DISCUSSION THREAD

**Discussion by ChenRuiwei** [flameshot-org/flameshot#3132] [Q&A]
I have no idea about why I can drag the pinned image while my mouse cursor isn't above the image itself. Is it a feature or a bug?
![recoder](https://user-images.githubusercontent.com/92102709/225825442-451d472c-5e18-4967-897a-becb3915c0e8.gif)

**Comment by mmahmoudian** [comment]
Interesting! What is your Flameshot version?

**Reply by ChenRuiwei** [reply]
I am an arch user, and I get flameshot by pacman. 
![2023-03-18_16-31](https://user-images.githubusercontent.com/92102709/226094708-5a716a4b-0009-4cc6-8d45-a66867fe5435.png)

**Reply by ChenRuiwei** [reply]
And this is the detailed system information
![2023-03-18_16-38](https://user-images.githubusercontent.com/92102709/226094988-c4f897e7-03ac-4a43-9bff-8e4e565cf73b.png)

**Reply by mmahmoudian** [reply]
I have the same setup (Plasma and X11 on Manjaro) and I don't have this issue. The only difference is that I have AMD instead of Nvidia:

```sh
❯ inxi --system --graphics --width 80
System:
  Host: precisiontower Kernel: 5.15.94-1-MANJARO arch: x86_64 bits: 64
    Desktop: KDE Plasma v: 5.26.5 Distro: Manjaro Linux
Graphics:
  Device-1: Intel CometLake-S GT2 [UHD Graphics 630] driver: i915 v: kernel
  Device-2: AMD Lexa XT [Radeon PRO WX 3200] driver: amdgpu v: kernel
  Display: x11 server: X.Org v: 21.1.7 driver: X: loaded: modesetting
    failed: amdgpu dri: radeonsi,iris gpu: amdgpu resolution: 1: 1920x1080~60Hz
    2: 1080x1920~60Hz 3: 1080x1920~60Hz
  API: OpenGL v: 4.6 Mesa 22.3.5 renderer: AMD Radeon Pro WX 3200 Series
    (polaris12 LLVM 15.0.7 DRM 3.42 5.15.94-1-MANJARO)
```

**Comment by ChenRuiwei** [comment]
I guess that's a problem caused by nvidia graphic card driver or the GraphicsProcessingUnitDirect technology. I am using a laptop and when I changed the GraphicsProcessingUnitDirect mode to hybrid graphics mode, the problem was solved.

**Reply by mmahmoudian** [reply]
The I also think the issue comes from either the graphics driver or Qt. Lets hope in the next versions they address this.
